x-pack/plugins/reporting/public/components/reporting_panel_content.tsx

```diff
@@ -40,24 +42,32 @@ class ReportingPanelContentUi extends Component<Props, State> {
 
     this.state = {
       isStale: false,
-      absoluteUrl: this.getAbsoluteReportGenerationUrl(props),
+      curlParams: this.getReportGenerationCurl(props),
       layoutId: '',
     };
   }
+  /*
+    curl --request POST 'http://u:p@localhost:5601/api/reporting/generate/printablePdf' \
+  --header 'Content-Type: application/json' \
```

Comment:
I pulled the PR and tested the curl command that it generated. In my testing, the command needed a few more arguments. The full command that worked for me was:

```
curl -X POST \
--location \
--header "kbn-xsrf: Reporting" \
--header "Content-Type: application/json" \
--data-raw "{\"jobParams\":\"(browserTimezone:America/Phoenix,layout:(dimensions:(height:2024,width:1979),id:preserve_layout),objectType:dashboard,relativeUrls:!('/app/dashboards#/view/722b74f0-b882-11e8-a6d9-e546fe2bba5f?_g=(filters:!!(),refreshInterval:(pause:!!f,value:900000),time:(from:now-7d,to:now))&_a=(description:!'Analyze%20mock%20eCommerce%20orders%20and%20revenue!',filters:!!(),fullScreenMode:!!f,options:(hidePanelTitles:!!f,useMargins:!!t),query:(language:kuery,query:!'!'),timeRestore:!!t,title:!'%5BeCommerce%5D%20Revenue%20Dashboard!',viewMode:view)'),title:'[eCommerce] Revenue Dashboard')\"}" \
http://localhost:5601/api/reporting/generate/printablePdf

```